Adyen/services.py

```diff
@@ -322,6 +329,42 @@ def payment_result(self, request=None, **kwargs):
         action = "paymentsResult"
         return self.client.call_checkout_api(request, action, **kwargs)
 
+    def payment_captures(self, path_param, request=None, idempotency_key=None, **kwargs):
+        if path_param == "":
+            raise ValueError(
+                'must contain a pspReference in the path_param, path_param cannot be empty'
+            )
+        action = "paymentsCapture"
```

Comment:
can you please add this action in the `client.py` code? I think it's not there yet